openpype/hosts/maya/plugins/publish/validate_frame_range.py

```diff
@@ -69,7 +73,7 @@ def process(self, instance):
         if [ef for ef in self.exclude_families
                 if instance.data["family"] in ef]:
             return
-        if(inst_start != frame_start_handle):
+        if (inst_start != frame_start_handle):
             errors.append("Instance start frame [ {} ] doesn't "
                           "match the one set on instance [ {} ]: "
                           "{}/{}/{}/{} (handle/start/end/handle)".format(
```

Comment:
Not introduced by this PR, but reading this message now I think one of the words "instance" should either be `context` or `asset`, right? Same for the one on line 85.

Reply:
Yup, done.